### PR TITLE
Update l2 s01e01 themes

### PR DESCRIPTION
**Pull request by IsaackMwenda**

### CODE REVIEW

File: code_schemes/s01e01.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
         "DisplayText": "meta: answer",
         "NumericValue": -1000140,
         "StringValue": "answer",
-        "VisibleInCoda": true,
+        "VisibleInCoda": false,
```
Comment (as2388) [inline code_schemes/s01e01.json:43]: Does this need a full deprecation or is it fine just to hide in Coda?

File: code_schemes/s01e01.json [hunk 2]
```diff
@@ -77,7 +77,7 @@
       },
       {
         "DisplayText": "yes, elderly at risk",
-        "VisibleInCoda": true,
+        "VisibleInCoda": false,
```
Comment (as2388) [inline code_schemes/s01e01.json]: Same here

File: code_schemes/s01e01.json [hunk 3]
```diff
@@ -147,6 +147,230 @@
...
+        "NumericValue": 15,
+        "StringValue": "symptoms",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-d606d1e8",
+        "CodeType": "Normal",
+        "DisplayText": "spread/transmission",
+        "NumericValue": 16,
+        "StringValue": "spread_transmission",
```
Comment (as2388) [inline code_schemes/s01e01.json:163]: I think this was called how spread/transmitted elsewhere, not sure if you need to change here.

File: code_schemes/s01e01.json [hunk 4]
```diff
@@ -147,6 +147,230 @@
...
+        "CodeType": "Normal",
+        "DisplayText": "yes, poor economy",
+        "NumericValue": 35,
+        "StringValue": "yes_poor_economy",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-7e6f682e",
+        "CodeType": "Normal",
+        "DisplayText": "yes, not following authorities advice",
```
Comment (as2388) [inline code_schemes/s01e01.json]:
```suggestion
        "DisplayText": "yes, not following authorities' advice",
```

**Review by as2388 — APPROVED** [review]

File: code_schemes/s01e01.json (hunk 1 above)
Comment (IsaackMwenda) [reply code_schemes/s01e01.json:43]: fine to hide in coda

File: code_schemes/s01e01.json (hunk 2 above)
Comment (IsaackMwenda) [reply code_schemes/s01e01.json]: ditto

File: code_schemes/s01e01.json (hunk 3 above)
Comment (IsaackMwenda) [reply code_schemes/s01e01.json:163]: adding it as is in the thematic document.

File: code_schemes/s01e01.json (hunk 4 above)
Comment (IsaackMwenda) [reply code_schemes/s01e01.json]: updated thanks

**Pull request merged**